lib/realtime/vehicle.ex

```diff
@@ -26,6 +27,13 @@ defmodule Realtime.Vehicle do
           direction_id: Direction.id(),
           route_id: Route.id(),
           trip_id: Trip.id(),
+          bearing: integer() | nil,
+          speed: integer() | nil,
+          stop_sequence: integer() | nil,
+          block_id: String.t() | nil,
+          operator_id: String.t() | nil,
+          operator_name: String.t() | nil,
+          run_id: String.t() | nil,
```

Comment:
What about instead of duplicating all the fields, you store the VehiclePosition and TripUpdate structs and look the values up as needed?

Reply:
What would be the benefit there? We'd need to convert to this Vehicle struct at some point as this is what we want to pass to the client. It feels like storing Vehicles keeps the realtime server code simpler, and limits the footprint of the interface between concentrate and the rest of the app.